src/GovUk.Education.ExploreEducationStatistics.Common.Tests/Utils/BoundaryDataUtilsTests.cs

```diff
@@ -0,0 +1,143 @@
+using GovUk.Education.ExploreEducationStatistics.Common.Utils;
+using System;
+using System.Collections.Generic;
+using Xunit;
+
+namespace GovUk.Education.ExploreEducationStatistics.Common.Tests.Utils;
+public class BoundaryDataUtilsTests
+{
+    [Fact]
+    public void GetCode_NoMatchingSuffix_ThrowsArgumentException()
```

Comment:
Not really a big deal but the new tests which have been written aren't following the backend styleguide point 3.1.1 which we came up with fairly recently which says

> Create a nested class on a per-public-method being tested basis {MethodUnderTest}Tests